<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreviSat</class>
 <widget class="QMainWindow" name="PreviSat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>684</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>684</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">PreviSat</string>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="layoutFenetre">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QFrame" name="frameCarteListe">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frameCarte">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="layoutCarte">
            <item>
             <widget class="QGraphicsView" name="graphicsView"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QFrame" name="frameCarteBoutons">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QToolButton" name="zoomCarte">
             <property name="geometry">
              <rect>
               <x>2</x>
               <y>0</y>
               <width>22</width>
               <height>22</height>
              </rect>
             </property>
             <property name="icon">
              <iconset resource="../../resources.qrc">
               <normaloff>:/resources/maxi.png</normaloff>:/resources/maxi.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QToolButton" name="changerCarte">
             <property name="geometry">
              <rect>
               <x>2</x>
               <y>25</y>
               <width>22</width>
               <height>22</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../resources.qrc">
               <normaloff>:/resources/globe.png</normaloff>:/resources/globe.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frameModeListe">
         <property name="minimumSize">
          <size>
           <width>239</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>239</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frameOngletsRadar">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>241</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>241</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frameOnglets">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frameRadar">
         <property name="minimumSize">
          <size>
           <width>279</width>
           <height>239</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>279</width>
           <height>239</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="barreMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>22</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuRaccourcis">
    <property name="title">
     <string notr="true">Raccourcis</string>
    </property>
    <addaction name="actionRaccourci_Previsions"/>
    <addaction name="actionRaccourci_Flashs"/>
    <addaction name="actionRaccourci_Transits"/>
    <addaction name="actionRaccourci_Evenements"/>
    <addaction name="separator"/>
    <addaction name="actionRaccourci_Informations"/>
    <addaction name="actionRaccourci_Recherche"/>
    <addaction name="actionRaccourci_Station"/>
   </widget>
   <addaction name="menuRaccourcis"/>
  </widget>
  <widget class="QStatusBar" name="barreStatut">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionRaccourci_Previsions">
   <property name="text">
    <string notr="true">Raccourci Previsions</string>
   </property>
   <property name="shortcut">
    <string notr="true">Alt+P</string>
   </property>
  </action>
  <action name="actionRaccourci_Flashs">
   <property name="text">
    <string notr="true">Raccourci Flashs</string>
   </property>
   <property name="shortcut">
    <string notr="true">Alt+F</string>
   </property>
  </action>
  <action name="actionRaccourci_Transits">
   <property name="text">
    <string notr="true">Raccourci Transits</string>
   </property>
   <property name="shortcut">
    <string notr="true">Alt+T</string>
   </property>
  </action>
  <action name="actionRaccourci_Evenements">
   <property name="text">
    <string notr="true">Raccourci Evenements</string>
   </property>
   <property name="shortcut">
    <string notr="true">Alt+E</string>
   </property>
  </action>
  <action name="actionRaccourci_Informations">
   <property name="text">
    <string notr="true">Raccourci Informations</string>
   </property>
   <property name="shortcut">
    <string notr="true">Alt+I</string>
   </property>
  </action>
  <action name="actionRaccourci_Recherche">
   <property name="text">
    <string notr="true">Raccourci Recherche</string>
   </property>
   <property name="shortcut">
    <string notr="true">Alt+R</string>
   </property>
  </action>
  <action name="actionRaccourci_Station">
   <property name="text">
    <string notr="true">Raccourci Station</string>
   </property>
   <property name="shortcut">
    <string notr="true">Alt+S</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
